plasmapy/diagnostics/tests/test_proton_radiography.py

```diff
@@ -417,6 +460,203 @@ def test_synthetic_radiograph():
     h, v, i = sim.synthetic_radiograph(size=size, bins=bins, optical_density=True)
 
 
+def test_gaussian_sphere_analytical_comparison():
+    """
+    This test runs a known example problem and compares to a theoretical
+    model for small deflections.
+
+    Still under construction (comparing the actual form of the radiograph
+    is possible but tricky to implement).
```

Comment:
This is a cool idea! I suspect with this sort of thing you'd need to run a bunch of simulations, then average the histograms to limit particle noise?

Reply:
Well, one simulation is sufficient with enough particles to limit noise, but there are also histogram-boxing effects that you have to be careful about. 

Really a spherical geometry isn't ideal here: the tests in `test_1D_deflections` are also comparisons against an analytical result (sort of) but since they are cylindrical it's easy to sum up one dimension and get a low-noise result with fewer particles.